config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => 'Free PMO',
+    'name' => 'Anchis & Kolbins',
```

Comment:
Please revert back this config `app.name` to **"Free PMO"**.

Reply:
Done